calibrations/tpc/gemcurrents/ConditionsForOneRun.sh

```
@@ -0,0 +1,11 @@
+#! /usr/bin/bash
+
+PROCESS=$1
+LINE=$((PROCESS + 1))
+RUNNUMBER=$(sed -n "${LINE}p" alltpc_runlist_PRUNED.txt)
+
+echo "Process: ${PROCESS}"
+echo "Run: ${RUNNUMBER}"
+
+python3 daqdbreader_tpc_GEM_charge_status_2.py ${RUNNUMBER}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)